### ISSUE TITLE
test  error

### ISSUE THREAD

**Issue by xiaosuzhang** [hujie-frank/SENet#56]
I have successfully trained the SE-Net, but when i use  Caffe Python wrapper to test the model, it will  encounter the following error and i don't konw how to solver this problem, can you help me,thanks:
] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Axpy (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, CenterLoss, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, Log, MVN, MarginInnerProduct, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)

**Comment by hujie-frank** [comment]
Please refer to the issue #35.

**Comment by DanChen001** [comment]
Hi, did you solve this problem?  thanks.  I meet the same problem.

**Comment by xibinyue** [comment]
我也碰到这个问题，训练的时候正常，但是python调用不正常
